test/functional/api/cas/cache.py

```diff
@@ -66,6 +65,22 @@ def get_status(self):
         status = self.get_statistics().config_stats.status.replace(' ', '_').lower()
         return CacheStatus[status]
 
+    @staticmethod
+    def get_metadata_max_size(cache_size: Size, cache_line_size: Size, number_of_cores=4096):
```

Comment:
I'd rename it to `calculate_metadata_max_size()` as this function is not a getter at all. It produces result based only on input numbers.